Sporadic build failure: TlsEnabledInProcPravegaClusterTest

**Problem description**
Encountered following test failure during travis build:
```
io.pravega.local.TlsEnabledInProcPravegaClusterTest > testWriteAndReadEventWhenConfigurationIsProper FAILED
    java.lang.AssertionError: expected:<Test event over TLS channel> but was:<null>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:118)
        at org.junit.Assert.assertEquals(Assert.java:144)
        at io.pravega.local.TlsEnabledInProcPravegaClusterTest.testWriteAndReadEventWhenConfigurationIsProper(TlsEnabledInProcPravegaClusterTest.java:121)
```

**Problem location**
TLS enabled cluster test

**Suggestions for an improvement**
Fix the test

--- a/standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java
+++ b/standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java
@@ -57,7 +57,7 @@ public class TlsEnabledInProcPravegaClusterTest extends InProcPravegaClusterTest
     // Note: Strictly speaking, this test is really an "integration test" and is a little
     // time consuming. For now, its intended to run as a unit test, but it could be moved
     // to an integration test suite if and when necessary.
-    @Test(timeout = 50000)
+    @Test(timeout = 200000)
     public void testWriteAndReadEventWhenConfigurationIsProper() throws ExecutionException,
             InterruptedException, ReinitializationRequiredException {
 

--- a/standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java
+++ b/standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java
@@ -115,7 +115,7 @@ public class TlsEnabledInProcPravegaClusterTest extends InProcPravegaClusterTest
 
         // Keeping the read timeout large so that there is ample time for reading the event even in
         // case of abnormal delays in test environments.
-        EventRead<String> event = reader.readNextEvent(50000);
+        EventRead<String> event = reader.readNextEvent(150000);
         String readMessage = event.getEvent();
         log.debug("Read event '{}", readMessage);
         assertEquals(message, readMessage);

